standalone/README.md

```diff
@@ -142,10 +159,15 @@ stringData:
   bucket: sdg-data
   access_key: *****
   secret_key: *****
-  data_key: sdg.tar.gz
+  data_key: data.tar.gz
 EOF
 
-./standalone run --namespace my-namespace --sdg-object-store-secret sdg-data
+./standalone run \
+  --namespace my-namespace \
+  --eval-serving-endpoint http://serving.kubeflow.svc.cluster.local:8080/v1 \
+  --eval-serving-model-name my-model \
+  --eval-serving-model-api-key ***** \
```

Comment:
canthis be `judge-serving` instead of `eval-serving`? ilab uses the term `judge_model`

Reply:
Addressed.

Reply:
```
--judge-serving-model-name
--judge-serving-endpoint
--judge-serving-model-api-key
```